examples/experiments/grounded_program_synthesis/train_trlx.py

```diff
@@ -59,12 +59,12 @@ def main(hparams={}):
     dataset = DSLDataset()
     train_prompts = list(dataset.load_datapoints(split="train"))[:1000]
 
-    model = trlx.train(
+    trainer = trlx.train(
         reward_fn=reward_fn,
         prompts=train_prompts,
         config=config,
     )
-    model.save_pretrained("dataset/trained_model")
+    trainer.save_pretrained("dataset/trained_model")
```

Comment:
Do the model types we use support save_pretrained?

Reply:
Yes

Reply:
Wait I don't think they do, or at least ppo doesn't.

Reply:
Ohhhh ok wait that's weird then can we just add a save pretrained function to PPO haha

Reply:
But anyway saving doesn't have anything to do with this pr so I think it's fine for now.